src/software/parameter/config/network.yaml

```diff
@@ -12,3 +12,26 @@ NetworkConfig:
       description: >-
           The network interface that is connected to the thunderbots router.
           Can be found using ifconfig on ubuntu.
+    VisionIP:
+      default: "224.5.23.2"
+      type: "std::string"
+      description: The IP address to listen on for vision data
+    VisionPort:
+      min: 1
+      max: 65535
+      default: 10020
+      type: "int"
+      description: The port to listen on for vision data
+    GamecontrollerIP:
```

Comment:
Going along with what @akhilveeraghanta 's comment said, consider changing to `GamecontrollerIPv4Address`

Reply:
done